deepfence_frontend/apps/dashboard/src/features/vulnerabilities/pages/RuntimeBom.tsx

```diff
@@ -210,76 +104,71 @@ const RuntimeBom = () => {
+            <DFLink to={'/vulnerability'} unstyled>
+              Vulnerabilities
+            </DFLink>
           </BreadcrumbLink>
           <BreadcrumbLink>
             <span className="inherit cursor-auto">Runtime BOM</span>
           </BreadcrumbLink>
         </Breadcrumb>
 
-        <span className="ml-2">
-          {navigation.state === 'loading' ? <CircleSpinner size="xs" /> : null}
+        <span className="ml-2 flex items-center">
+          {isFetching ? <CircleSpinner size="sm" /> : null}
         </span>
       </div>
-      <div className="m-2">
-        <Suspense fallback={<TableSkeleton columns={2} rows={15} size={'md'} />}>
-          <DFAwait resolve={loaderData.scans}>
-            {(resolvedData: LoaderData['scans']) => {
-              return (
-                <Table
-                  size="sm"
-                  data={resolvedData.scans}
-                  columns={columns}
-                  enablePagination
-                  manualPagination
-                  enableColumnResizing
-                  totalRows={resolvedData.totalRows}
-                  pageSize={PAGE_SIZE}
-                  pageIndex={resolvedData.currentPage}
-                  onPaginationChange={(updaterOrValue) => {
-                    let newPageIndex = 0;
-                    if (typeof updaterOrValue === 'function') {
-                      newPageIndex = updaterOrValue({
-                        pageIndex: resolvedData.currentPage,
-                        pageSize: PAGE_SIZE,
-                      }).pageIndex;
-                    } else {
-                      newPageIndex = updaterOrValue.pageIndex;
-                    }
-                    setSearchParams((prev) => {
-                      prev.set('page', String(newPageIndex));
-                      return prev;
-                    });
-                  }}
-                  enableSorting
-                  manualSorting
-                  sortingState={sort}
-                  onSortingChange={(updaterOrValue) => {
-                    let newSortState: SortingState = [];
-                    if (typeof updaterOrValue === 'function') {
-                      newSortState = updaterOrValue(sort);
-                    } else {
-                      newSortState = updaterOrValue;
-                    }
-                    setSearchParams((prev) => {
-                      if (!newSortState.length) {
-                        prev.delete('sortby');
-                        prev.delete('desc');
-                      } else {
-                        prev.set('sortby', String(newSortState[0].id));
-                        prev.set('desc', String(newSortState[0].desc));
-                      }
-                      return prev;
-                    });
-                    setSort(newSortState);
-                  }}
-                />
-              );
+      <div className="m-4">
+        <Suspense fallback={<TableSkeleton columns={2} rows={10} />}>
```

Comment:
Suspense does not work in the same component as react query, should be wrapped in parent.